docs/application/native/guides/app-management/sticker.md

```diff
@@ -0,0 +1,431 @@
+# Sticker
+
+
+The Sticker feature provides the functions to manage and retrieve sticker information. The Sticker application is divided into the [Sticker provider application](#provider_application) that creates the sticker and the [Sticker consumer application](#consumer_application) that uses the sticker created by the Sticker provider. The Sticker provider application can store sticker details such as URI, keyword, and group name to the Sticker database. The Sticker consumer application can retrieve sticker information using the Sticker API.
+
+The main features of the Sticker API include:
+
+- Preparing the Sticker service for use
+
+  You can connect the Sticker daemon to be able to operate the Sticker service.
+
+- Sticker data handle
+
+  You can use the Sticker data handle to [store the sticker information](#set_data_handle) and to [get the sticker information](#get_data_handle) stored in it.
+
+- [Inserting the sticker information](#insert_sticker)
+
+  - You can store sticker information using the Sticker API at application runtime, and [use JSON file](#set_json) to store many sticker information at once.
+  - You can store sticker information automatically when the Sticker provider application is installed without using the Sticker API.
+
+- Retrieving sticker information
+
+  You can [retrieve sticker information](#retrieve_sticker), which includes URI, keyword, group name and so on.
+
+**Figure: Sticker mechanism**
+
+![Sticker mechanism](./media/sticker.png)
+
```

Comment:
Suggestion.
I didn't finish my review yet, but before other comments, I have a suggestion.

How about taking one "Prerequisites" section only as below?

## Prerequisites

To enable your Sticker provider application to use the Sticker functionality:

1. To use the Sticker provider API, the following category must be added to the `tizen-manifest.xml` file:

    ```
    <category name="http://tizen.org/category/sticker"/>
    ```

2. To use the functions and data types of the Sticker API, include the `<sticker_provider.h>` header file in your application:

    ```
    #include <sticker_provider.h>
    ```

To enable your Sticker consumer application to use the Sticker functionality:

1. To use the Sticker consumer API, the application has to request permission by adding the following privilege to the `tizen-manifest.xml` file:

    ```
    <privileges>
       <privilege>http://tizen.org/privilege/mediastorage</privilege>
    </privileges>
    ```

2. To use the functions and data types of the Sticker API, include the `<sticker_consumer.h>` header file in your application:

    ```
    #include <sticker_consumer.h>
    ```

 To ensure that a Sticker function has been executed properly, make sure that the return is equal to `STICKER_ERROR_NONE`.


## Creating Sticker provider application
...
## Creating Sticker consumer application
...

Reply:
Fixed.